Firebase/Messaging/FIRMessagingRemoteNotificationsProxy.m

```diff
@@ -32,7 +34,11 @@
 static NSString *kUserNotificationDidReceiveResponseSelectorString =
     @"userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:";
 
+#if TARGET_OS_IOS ||TARGET_OS_TV
 @interface FIRMessagingRemoteNotificationsProxy () <UIApplicationDelegate>
+#else
+@interface FIRMessagingRemoteNotificationsProxy ()
+#endif
```

Comment:
Since #2911 landed this can be replaced by `@interface FIRMessagingRemoteNotificationsProxy () <GULApplicationDelegate>`

Reply:
Done!